src/models/team.rs

```diff
@@ -1,22 +1,54 @@
 use crate::schema::teams;
 use chrono::NaiveDateTime;
+use regex::Regex;
 use serde::{Deserialize, Serialize};
 
+use rand::prelude::*;
+
+lazy_static! {
+    static ref SLUG_REGEX: Regex = Regex::new("^[a-z0-9\\-]+$").unwrap();
```

Comment:
regex should only be a-z0-9_-
```suggestion
    static ref SLUG_REGEX: Regex = Regex::new("^[a-z0-9-_]+$").unwrap();
```

Reply:
Why? The current regex matches lowercase characters + numbers + hyphens

Reply:
it actually matches `\` as well which may break it

Reply:
It actually doesn't, the `\\` is just an escape character for the `-` (two are needed so Rust interprets it as a literal `\`)